packages/tabs/src/Tab.tsx

```diff
@@ -0,0 +1,48 @@
+import { Tab as ReachTab } from "@reach/tabs"
+import React, { ReactNode, useState } from "react"
+import classnames from "classnames"
+import { Badge } from "@kaizen/draft-badge"
+import styles from "./styles.scss"
+
+export interface TabProps {
+  /**
+   * Gets injected by TabList, no need to specify yourself
+   */
+  isSelected?: boolean
+  /**
+   * Adds a Kaizen Badge component to the tab.
+   * Comes with some logic baked in - changes variant based on active/focus/hover state.
+   */
+  badge?: string
+  disabled?: boolean
+  children: ReactNode
```

Comment:
We might want to pre-emptively expose a couple of events for tabs - I imagine folks would want to use them for analytics and such. We expose `onClick`, `onBlur`, `onMouseDown` and `onFocus` for button.

Reply:
I've done this but left out `onMouseDown` because I'm not sure I want to be encouraging people to do things exclusive to mouse users.